web_api/app/precomputed_annotations.py

```diff
@@ -38,12 +38,14 @@ async def read_in_bounds(
     resolution: Annotated[tuple[float, float, float], Query()],
 ):
     """
-    This endpoint retrieves all lines within the given bounds.
+    This endpoint retrieves all lines entirely within the given bounds.
+    Coordinates are returned in units according to `resolution`.
     """
-    index = VolumetricIndex.from_coords(bbox_start, bbox_end, Vec3D(*resolution))
+    resolution_vec = Vec3D(*resolution)
+    bbox = BBox3D.from_coords(bbox_start, bbox_end, resolution_vec)
     layer = build_annotation_layer(path, mode="read")
     response = []
-    for line in layer.read_in_bounds(index):
+    for line in layer.read_in_bounds(bbox, strict=True, annotation_resolution=resolution_vec):
```

Comment:
Should we decouple the bbox specification resolution from the resolution we want the resulting annotations to be in?

Reply:
I'd say that's up to the users of the web API (your team in Ukraine, I guess).  Most of the time I would expect them to be the same; it's also easier to avoid mistakes if you don't have to keep two different resolutions in mind.  But if they have some reason to need them separate, we can easily do that.

Reply:
It's just a cleaner more modern way to use python that reduces boilerplate and comes with a lot of free gooddies